src/main/java/net/fabricmc/loader/metadata/CustomValueImpl.java

```diff
@@ -185,6 +237,56 @@ public CustomValue get(int index) {
 		public Iterator<CustomValue> iterator() {
 			return entries.iterator();
 		}
+
+		@Override
+		public Stream<CustomValue> stream() {
+			return StreamSupport.stream(this.spliterator(), false);
```

Comment:
Couldn't you get the stream from the list that backs the array impl?

Reply:
i can, didn't notice that